docs/development/design/CasualtySelection.puml

```diff
@@ -0,0 +1,61 @@
+@startuml
```

Comment:
I'd definitely suggest some design documentation and wanted to encourage that by starting.
The more fundamental the design documentation the longer it will be meaningful.
```UnitChooser.pml``` describes a structure that IMHO should be redesigned (by somebody who doesn't cause the main programmer so much hassle with a minor change - thanks for your patience with me :) ). I hope the diagram will be helpful then (and I also hope it can the be removed or replaced with something showing a nicer design).
```CasualtySelection.pml``` lists the use cases to be restested when casualty selection is being changed. I suspect that will happen once or twice in the near future. The design docu should then be useful. If you agree that it should be but think that it isn't, I can happily try to improve it.
If you agree that one or both design docs should be added, I happily include information how to use them in the README. I would be even happier, if someone with a greater understanding of the whole system would write something about design documentation at the appropriate place and then maybe refer to my diagrams as sort of a seed effort.

Reply:
I'll leave it to your judgement if you think they will be worth keeping.
If so, needs to be done well. Notably a mention somewhere in the dev documentation of how to view and render these diagrams.
There is also a 'docs/background/design' folder as well, no need to create another 'design/' folder

Reply:
The next commit will move the improved files, which will include a brief info how to view them within the files.